test/test_nn.py

```diff
@@ -18295,6 +18295,59 @@ def test_cross_entropy_label_smoothing_with_probs(self, device):
                 self.assertEqual(output_with_smoothing, output_with_manual_smoothing)
 
 
+    def test_cross_entropy_label_smoothing_weight_ignore_indices(self, device):
+        reductions = ['none', 'sum', 'mean']
+        label_smoothings = [0.05, 0.15]
```

Comment:
FYI for future reference: I added a feature to obviate the need for manually computing the product over params in cases like this. You can decorate the test as so and it generates tests across the product for you:

```python
@parametrize_test('reduction', ['none', 'sum', 'weight'])
@parametrize_test('label_smoothing', [0.05, 0.15])
def test_cross_entropy_label_smoothing_weight_ignore_indices(self, device, reduction, label_smoothing)
    ...
```

I won't hold up this PR to change this but wanted you to know in case it makes your life easier later on :)